code/modules/clothing/head/misc_special.dm

```diff
@@ -41,12 +41,12 @@
 			src.up = !src.up
 			src.body_parts_covered = FACE
 			eyeprot = 3
-			icon_state = initial(icon_state)
```

Comment:
Leave this and the one below as they were, for future compatibility

Reply:
sadly I can't, since im starting with "weldingup" as icon_state. this would mean if its down icon_state would be "weldingup" and when its up "weldingupup"

Only workaround I though of was to set it "welding" initally and redeclare it in /New() to "weldingup", but someone in discord told me its ok to just put the string.

Unless you may have a way to strip "up" from the string, I might have to leave it this way.

Reply:
Fair point, I didn't think of that. Does the welding helmet have any children? If they inherit the same procs this may break their icon when flipped, if not all is good

Reply:
nope, didn't find any children

Reply:
All good then, good job